xml/security_ldap_replication.xml

```diff
@@ -0,0 +1,469 @@
+    <listitem>
+     <para>
+      A replica that receives data from another replica to write into its
+      database.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replication agreement</term>
+    <listitem>
+     <para>
+      The configuration of a server defining its supplier and consumer relation
+      to another replica.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Topology</term>
+    <listitem>
+     <para>
+      A set of replicas connected via replication agreements.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replica ID</term>
+    <listitem>
+     <para>
+      A unique identifier of the &ds389; instance within the replication
+      topology.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replication manager</term>
+    <listitem>
+     <para>
+      An account with replication rights in the directory.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect2>
+
+ <sect2 xml:id="sec-security-ldap-replication-configuration">
+  <title>Configuring replication</title>
+  <para>
+   The first example sets up a two node bi-directional replication with a
+   single read-only server, as a minimal starting example. In the following
+   examples, the host names of the two read-write nodes are RW1 and RW2, and
+   the read-only server is RO1. (Of course you must use your own host names.)
+  
+   </para>
+  <para>
+   All servers should have a backend with an identical suffix. Only one server,
+   RW1, needs an initial copy of the database.
+  </para>
+  <para>
+   The following commands configure the read-write replicas, RW1 and RW2.
+   First, configure RW1:
+  </para>
+<screen>&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication create-manager</command>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication enable</command> \ 
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \
+<command>--role supplier --replica-id 1 --bind-dn "cn=replication manager,cn=config"</command>
+</screen>
+  <para>
+   Configure RW2:
+  </para>
+<screen>&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication create-manager</command>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication enable</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \ 
+<command>--role supplier --replica-id 2 --bind-dn "cn=replication manager,cn=config"</command>
+</screen>
+  <para>
+   This will create the replication metadata required on RW1 and RW2. Note the
+   difference in the <option>replica-id</option> between the two servers. This
+   also creates the replication manager account, which is an account with
+   replication rights for authenticating between the two nodes.
```

Comment:
Should add a comment that replication manager should be considered equivalent to directory manager in terms of security and "unrestricted access" so must use a strong random password (IE `pwgen 40`)